src/super_gradients/training/models/detection_models/yolox.py

```diff
@@ -0,0 +1,50 @@
+from super_gradients.training.models.detection_models.yolov5_base import YoLoV5Base, YoLoV5DarknetBackbone
+from super_gradients.training.utils.utils import HpmStruct
+
+
+class YoloX_N(YoLoV5Base):
```

Comment:
why do we need all of these if we have yamls of them? why not just define YoloX and let the yaml do the rest